ChunkyDocs/mkdocs.yml

```diff
@@ -101,49 +109,52 @@ nav:
   - "Home": "index.md"
   - "Getting Started":
     - "Installing Chunky": "getting_started/installing_chunky.md"
-    - "Configuring Chunky Launcher": "getting_started/configuring_chunky_launcher.md"
+    - "Configuring the Chunky Launcher": "getting_started/configuring_chunky_launcher.md"
     - "Your First Render": "getting_started/your_first_render.md"
   - "Support":
     - "FAQ": "support/faq.md"
     - "Troubleshooting": "support/troubleshooting.md"
     - "Minecraft Compatibility": "support/minecraft_compatibility.md"
+  - "User Guides":
```

Comment:
I do not believe that 'User Guides' should come before 'Reference'. The site flow feels really off. Surely we need to cover the 'Introduction' content before something like 'Denoising' makes sense? And wouldn't 'Introduction' be better suited to be part of  'User Guides'? Other topics in 'User Guides' like 'Skymaps' also don't make a huge amount of sense to introduce before we cover them.

Reply:
I moved the 'Introduction' articles into 'User Guides' so that samples and noise is covered before denoising.